.github/workflows/ci-tests-fabric.yml

```diff
@@ -108,6 +108,7 @@ jobs:
 
     - name: Install package & dependencies
       run: |
+        python -m pip install -q pip -U
```

Comment:
```suggestion
        pip install -q pip -U
```

Reply:
The error was because we didn't use `python -m`, that's the fix.